packages/config/src/api/site_info.ts

```diff
@@ -162,8 +125,8 @@ const getIntegrations = async function ({
 
   const baseUrl = new URL(host ? `http://${host}` : `https://api.netlifysdk.com`)
 
-  // use future state feature flag
-  const url = useV2Endpoint
+  // if accountId isn't present, use safe v1 endpoint
```

Comment:
Is this for backwards compatibility? I was expecting us to fully switch over to the new endpoint without the feature flag, but is it the case that there might still something still calling this code without an account id once this code goes out?

Reply:
Honestly this is just because I don't know if there would ever be a case where accountId isn't present. 😎 The config package has a few mysterious optional types on things like `accountId` where I'd anticipate them to always be present, but there's no comments around why they were marked as optional.

Reply:
That's fair enough. I was maybe being a little greedy thinking that we might even be able to get rid of the whole api endpoint for `${baseUrl}site/${siteId}/integrations/safe` in the other system. No change needed here, I was just learning.